go/preload/main/preload.go

```diff
@@ -147,7 +148,7 @@ func main() {
 		}
 	}()
 
-	fetchLogClient := client.New(*sourceLogUri)
+	fetchLogClient := client.New(*sourceLogUri, &http.Client{})
```

Comment:
To preserve the semantics, this should now set up the transport in a similar way to what was done before?